foundation-ui/factories/ServiceFactory.js

```diff
@@ -0,0 +1,23 @@
+/*
+  Service Factory
+ */
+function ServiceFactory(hooks) {
```

Comment:
It doesn't need to have 'factory' methods. This is a factory that creates services. See here for more examples:
https://medium.com/javascript-scene/javascript-factory-functions-vs-constructor-functions-vs-classes-2f22ceddf33e#.49vxo3zg0